media/stitcher/api/Stitcher.Samples.Tests/UpdateCdnKeyTests.cs

```diff
@@ -33,17 +33,9 @@ public UpdateCdnKeyTest(StitcherFixture fixture)
         [Fact]
         public void UpdatesCdnKey_CloudCdn()
         {
-            var result = _updateSample.UpdateCdnKey(_fixture.ProjectId, _fixture.LocationId, _fixture.TestCloudCdnKeyId, _fixture.UpdateHostname, _fixture.CloudCdnKeyName, _fixture.UpdatedCloudCdnTokenKey, null);
+            var result = _updateSample.UpdateCdnKey(_fixture.ProjectId, _fixture.LocationId, _fixture.TestCloudCdnKeyId, _fixture.UpdatedHostname, _fixture.KeyName, _fixture.CloudCdnPrivateKey, false);
             Assert.Equal(_fixture.TestCloudCdnKeyId, result.CdnKeyName.CdnKeyId);
-            Assert.Equal(_fixture.UpdateHostname, result.Hostname);
-        }
-
-        [Fact]
-        public void UpdatesCdnKey_Akamai()
```

Comment:
It's the media one not testable?

Reply:
Good catch. I added the test for Media CDN.